phone-scripts/02-install-packages.sh

```diff
@@ -16,32 +16,34 @@ errecho () {
 
 infecho "This adds my COPR repository (njha/mobile) and installs phone related packages."
 infecho "Only functional on Fedora Rawhide."
-infecho "HEAVY WIP, untested"
-
-infecho "Enabling COPR repository..."
-dnf copr enable njha/mobile
+dnf copr enable njha/mobile -y
 
 infecho "Removing old kernel..."
-infecho "THIS WILL FAIL, DON'T WORRY ITS PROBABLY OK"
-dnf remove kernel || rpm -e --noscripts kernel-core
-dnf install linux-firmware
+dnf remove kernel -y || rpm -e --noscripts kernel-core
 
 infecho "Installing kernel..."
-dnf --disablerepo="*" --enablerepo="copr:copr.fedorainfracloud.org:njha:mobile" install kernel
+dnf --disablerepo="*" --enablerepo="copr:copr.fedorainfracloud.org:njha:mobile" install megi-kernel-bh -y
 
 infecho "Installing recommended packages..."
-dnf install feedbackd phoc phosh squeekboard gnome-shell ModemManager rtl8723cs-firmware \
-    f2fs-tools chatty calls carbons purple-mm-sms pinephone-helpers evolution-data-server \
-    f32-backgrounds-gnome kgx epiphany gnome-contacts evolution cheese NetworkManager-wwan \
-    lightdm-mobile-greeter
-
-infecho "Enabling graphical boot and lightdm..."
+dnf install feedbackd phoc phosh squeekboard rtl8723cs-firmware	\
+    f2fs-tools chatty calls carbons purple-mm-sms pinephone-helpers \
+    f32-backgrounds-gnome kgx epiphany gnome-contacts NetworkManager-wwan \
+    ModemManager gnome-shell -y
+
+infecho "Removing unneeded packages..."
+dnf remove grub2 -y
+rm -rf rootfs/boot/grub2
+rm -rf rootfs/etc/grub.d
+rm rootfs/etc/grub2-efi.cfg
+
+infecho "Enabling phosh boot..."
+systemctl disable gdm.service
 systemctl disable initial-setup.service
-systemctl enable lightdm
```

Comment:
I think we want to keep lightdm so as to be able to switch to other environments such as GNOME.